static/js/components/forms/CouponForm.js

```diff
@@ -236,16 +247,15 @@ export const CouponForm = ({
             name="products"
             valueKey="id"
             labelKey="title"
-            options={products}
-            value={selectedProducts}
+            options={_.filter(products, { product_type: values.product_type })}
```

Comment:
Could you use ramda instead of pulling in lodash? `filter(products, whereEq(pick(['product_type], values)))` would probably do the trick

Reply:
Okay, though personally I prefer lodash over ramda,
```
_.filter(products, { product_type: values.product_type })
```
seems much less obfuscated to me than
```
{filter(pathSatisfies(equals(values.product_type), ['product_type']), products)}
```